### PR TITLE
Charged hadron muon ref filter

### PR DESCRIPTION
**Pull request by clelange**
This is an additional MET filter that vetoes events that contain charged hadrons with tracks that have bad quality. Fixed in 76X.

### CODE REVIEW

**Comment by slava77** [comment]
currently everything modifying the standard sequences in a noticeable way (of what's stored/produced in RECO/MINIAOD) and submitted to 74X is not going anywhere (can't call it /dev/null just yet; PRs are left unattended)

File: RecoMET/METFilters/python/metFilters_cff.py [hunk 1]
```diff
@@ -61,7 +68,9 @@
 #   hcalLaserEventFilter *
    EcalDeadCellTriggerPrimitiveFilter* 
 #   *goodVertices * trackingFailureFilter *
-   eeBadScFilter
+   eeBadScFilter *
 #   ecalLaserCorrFilter *
 #   trkPOGFilters
+   chargedHadronTrackResolutionFilter *
```
Comment (slava77) [inline RecoMET/METFilters/python/metFilters_cff.py]: changes in this file should not be made in 74X, until changes in RECO/miniAOD workflows are allowed.

**Comment by slava77** [comment]
@clelange @schoef @mariadalfonso 
do we really need this in 74X?

**Comment by clelange** [comment]
Since 7_4_X won't be used for any campaigns, I'd say this PR does not have to enter it - and it's already in 76X and I will make another request for 80X. Will close, can reopen in case there are different opinions.

**Pull request closed without merging**